__tests__/app.test.js

```diff
@@ -57,8 +57,69 @@ describe("/api", () => {
       .get("/api")
       .then(({ body }) => {
         const { endpoints } = body;
-        console.log(localEndPointsJSON)
-        expect(endpoints).toEqual(localEndPointsJSON)
+        expect(endpoints).toEqual(localEndPointsJSON);
+      });
+  });
+});
+
+describe("/api/articles/:article_id", () => {
+  test("200: returns an article object with the correct properties for a specific article_id", () => {
+    return request(app)
+      .get("/api/articles/4")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toHaveProperty("author", "rogersop");
+        expect(article).toHaveProperty("title", "Student SUES Mitch!");
+        expect(article).toHaveProperty("article_id", 4);
+        expect(article).toHaveProperty(
+          "body",
+          "We all love Mitch and his wonderful, unique typing style. However, the volume of his typing has ALLEGEDLY burst another students eardrums, and they are now suing for damages"
+        );
+        expect(article).toHaveProperty("topic", "mitch");
+        expect(article).toHaveProperty(
+          "created_at",
+          "2020-05-06T01:14:00.000Z"
+        );
+        expect(article).toHaveProperty("votes", 0);
+        expect(article).toHaveProperty(
+          "article_img_url",
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+        );
+      });
```

Comment:
If you wanted to tidy this up a little bit you could use toEqual, toMatchObject or ObjectContaining instead. Not necessary to change but worth having a look if you like!